pages/settings/wallets/lnbits.js

```diff
@@ -51,12 +51,11 @@ export default function LNbits () {
           required
           autoFocus
         />
-        <ClientInput
+        <PasswordInput
```

Comment:
Is the lnbits admin key required? I couldn't parse whether this needs to be required or not from the validation

Reply:
Oh, must have been an oversight when I implemented this. It's definitely required.